app/components/intermission.jsx

```diff
@@ -0,0 +1,202 @@
+// https://github.com/EnCiv/civil-pursuit/issues/137
```

Comment:
Please remove these parts:
<img width="405" alt="image" src="https://github.com/user-attachments/assets/f07b4432-f18e-4bc2-96df-3f947252b441">
I know it's in the figma, but we figured out its not necessary because they had to agree at the beginning.

Reply:
Still need to remove the part next to the button that says "Already have a password".